translations/ui-orders/en.json

```diff
@@ -726,8 +741,12 @@
   "settings.setPOLInesLimit": "Set purchase order lines limit",
 
   "settings.acquisitionMethods": "Acquisition methods",
-  "settings.acquisitionMethods.singular": "Acquisition method",
```

Comment:
You can't remove this. It's still in use via [the `objectLabel` prop](https://github.com/folio-org/ui-orders/blob/f0194a92d981635e8eec9496d99fd299d0a67a94/src/settings/AcquisitionMethods/AcquisitionMethods.js#L55)

Reply:
You are absolutely right, thanks! Updated